components/server/src/redis/mutex.ts

```diff
@@ -20,7 +20,7 @@ export class RedisMutex {
 
             // The max number of times Redlock will attempt to lock a resource
             // before erroring.
-            retryCount: 3,
+            retryCount: 20,
```

Comment:
Do I understand correctly that this changes the max. delay introduced by Redis, whenever we try to get a mutex, from 0.6s to 4s?
What's the motivation for the change? :thinking: